src/Symfony/Component/Security/Core/Tests/Encoder/Argon2iPasswordEncoder.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Core\Tests\Encoder;
+
+use Symfony\Component\Security\Core\Encoder\Argon2iPasswordEncoder;
+
+/**
+ * @author Elnur Abdurrakhimov <elnur@elnur.pro>
+ */
+class Argon2iPasswordEncoderTest extends \PHPUnit_Framework_TestCase
+{
+    const PASSWORD = 'password';
+
+    public function setUp()
+    {
+        if (PHP_VERSION_ID < 70200 && !extension_loaded('libsodium')) {
```

Comment:
One solution could be to check the result of `ReflectionFunctionAbstract::isUserDefined`, though would this be adding too much overhead?